integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdownMS.java

```diff
@@ -27,24 +25,13 @@
 /**
  * Simple JUnit test file used for testing Operator.
  *
- * <p>This test is used for testing pods being shutdowned by some properties change.
+ * <p>This test is used for applying Shutdown Properties at ManagedServer(MS) level.
  */
 @TestMethodOrder(Alphanumeric.class)
-public class ItPodsShutdown extends BaseTest {
+public class ItPodsShutdownMS extends ShutdownOptionsBase {
```

Comment:
I really dislike test classes that extend other classes, it make the behavior harder to understand and debug.

Reply:
Base contains only shared between two classes methods, not generic to put to BaseTest.java or TestUtils.java

Reply:
can you find a better way that does not use inheritence?